apps/web/env.d.ts

```diff
@@ -1,11 +1,19 @@
-// Generated by Wrangler on Sun May 26 2024 17:03:05 GMT-0500 (Central Daylight Time)
+// Generated by Wrangler on Thu Jul 25 2024 01:03:57 GMT-0700 (Pacific Daylight Time)
 // by running `wrangler types --env-interface CloudflareEnv env.d.ts`
 
 interface CloudflareEnv {
+	CANVAS_SNAPS: KVNamespace;
+	RECOMMENDATIONS: KVNamespace;
+	AUTH_URL: string;
+	AUTH_SECRET: string;
+	AUTH_GOOGLE_ID: string;
+	AUTH_GOOGLE_SECRET: string;
+	DATABASE_URL: string;
+	BACKEND_SECURITY_KEY: string;
+	BACKEND_BASE_URL: string;
```

Comment:
@Dhravya Can you run cf-typegen to regenerate this file using your account? Why are some env vars in `apps/web/cf-env.d.ts`?

Reply:
i've stopped doing cf-type gen and now just add stuff manually from CloudflareEnv

Reply:
moved it to cf-env